### PR TITLE
Admin interactivity

### PR DESCRIPTION
**Pull request by 0xDCA**
This PR adds basic interactivity for admin statistics. The list of relevant submissions will be shown when clicking on the relevant part of the plot.

### CODE REVIEW

File: inginious/frontend/webapp/plugins/statistics/static/js/course_admin_statistics.js [hunk 1]
```diff
@@ -129,6 +130,21 @@
...
+
+            container.unbind('plotly_click');
+            container[0].on('plotly_click', function(data) {
+                var point = data.points[0];
+                var taskId = point.data.taskId;
+
+                $.get('/api/stats/admin/grade_distribution_details', {
+                    course_id: adminStatistics.courseId,
+                    task_id: taskId
+                }, function(result) {
```
Comment (amrondonp) [inline inginious/frontend/webapp/plugins/statistics/static/js/course_admin_statistics.js:147]: Do we need to handle the case of error in the API call?

Comment (0xDCA) [reply inginious/frontend/webapp/plugins/statistics/static/js/course_admin_statistics.js:147]: I guess we can show an error message. Do you agree?

Comment (amrondonp) [reply inginious/frontend/webapp/plugins/statistics/static/js/course_admin_statistics.js:147]: Show it how, an alert?

File: inginious/frontend/webapp/plugins/statistics/utils.py [hunk 2]
```diff
@@ -41,3 +41,26 @@ def generate_task_element(task):
...
+
+def project_submission(submission):
+    if submission is None:
+        return None
+
+    return {
+        "id": str(submission["_id"]),
+        "submitted_on": submission["submitted_on"].isoformat(),
+        "taskId": submission["taskid"],
+        "status": submission["status"],
```
Comment (Maikol1804) [inline inginious/frontend/webapp/plugins/statistics/utils.py:62]: Is it important to show this status?

File: inginious/frontend/webapp/plugins/statistics/static/js/course_admin_statistics.js (hunk 1 above)
Comment (0xDCA) [reply inginious/frontend/webapp/plugins/statistics/static/js/course_admin_statistics.js:147]: Yes. I modified the code

File: inginious/frontend/webapp/plugins/statistics/utils.py (hunk 2 above)
Comment (0xDCA) [reply inginious/frontend/webapp/plugins/statistics/utils.py:62]: I guess it might be useful for grader failures or similar situations

**Comment by amrondonp** [comment]
LGTM

**Pull request merged**